Source/bmalloc/Configurations/modulemap.toml

```diff
@@ -0,0 +1,10 @@
+module-name = 'bmalloc'
+requirements = ['cplusplus11']
+
+[module.bexport]
+attributes = ['extern_c']
+headers = ['BExport.h']
+
+[module.pas_allocation_mode]
```

Comment:
Ditto about it

Reply:
Ditto, please add comments about why we need this separately, and when we need these things.

Reply:
I'll move the comment "can be used in plain C contexts" down a few lines to make it more obvious that it applies to this line.